samples/matter/lock/sample.yaml

```diff
@@ -60,3 +60,25 @@ tests:
     integration_platforms:
       - nrf52840dk_nrf52840
     platform_allow: nrf52840dk_nrf52840 nrf5340dk_nrf5340_cpuapp
+  sample.matter.lock.release.ffs:
```

Comment:
Use integration platforms argument to choose for which platform it should build, here and for other configurations where integration_platforms is missing.